index.js

```diff
@@ -31,10 +52,11 @@ module.exports = {
         }
       };
 
-      return new TemplateLinter(tree, {
+      return TemplateLinter.create(tree, {
         annotation: 'TemplateLinter',
         templatercPath: this.project.root + '/.template-lintrc',
-        generateTestFile: this.project.generateTestFile,
+        testGenerator: this._options.testGenerator,
+        group: (this._options.group !== false) ? type : undefined,
```

Comment:
there is a difference between the `group` option of the broccoli plugin and the `group` option of the Ember CLI addon. the former accepts a string to name the group, the latter only accepts a boolean to enable/disable grouping in general.